openedx/core/djangoapps/user_authn/views/login.py

```diff
@@ -56,27 +56,17 @@ def _do_third_party_auth(request):
             u"with backend_name {backend_name}".format(
                 username=username, backend_name=backend_name)
         )
-        message = _(
+        message = Text(_(
             u"You've successfully logged into your {provider_name} account, "
-            u"but this account isn't linked with an {platform_name} account yet."
-        ).format(
-            platform_name=platform_name,
-            provider_name=requested_provider.name,
-        )
-        message += "<br/><br/>"
-        message += _(
+            u"but this account isn't linked with {platform_name} account yet. {blank_lines}"
             u"Use your {platform_name} username and password to log into {platform_name} below, "
-            u"and then link your {platform_name} account with {provider_name} from your dashboard."
-        ).format(
-            platform_name=platform_name,
-            provider_name=requested_provider.name,
-        )
-        message += "<br/><br/>"
-        message += Text(_(
+            u"and then link your {platform_name} account with {provider_name} from your dashboard.{blank_lines}"
             u"If you don't have an {platform_name} account yet, "
```

Comment:
You could either leave this, or make it something like:
```
If you don't have an account on {platform_name} yet,
```
Note: this sentence is a little strange if you use "your", but that is a possibility.  I don't know who should make the call around wording.

Reply:
@robrap Done.